src/main/scala/com/typesafe/sbt/packager/linux/RpmNoReplaceConfigPlugin.scala

```diff
@@ -0,0 +1,18 @@
+package com.typesafe.sbt.packager.linux
+
+import com.typesafe.sbt.packager.archetypes.JavaServerAppPackaging
+import com.typesafe.sbt.packager.rpm.RpmPlugin
+import com.typesafe.sbt.packager.rpm.RpmPlugin.autoImport
+import sbt._
+
+/**
+  * This plugin automatically updates all config files in a RPM to "noreplace". Those files will not be overwritten
+  * during an update if they have been changed on disk. This is useful for server apps.
+  */
+object RpmNoReplaceConfigPlugin extends AutoPlugin with LinuxKeys with LinuxMappingDSL {
```

Comment:
Yep. When you have added some docs this is ready to be merged. Thanks for your time and patience :)